pic-sure-resources/pic-sure-resource-api/src/main/java/edu/harvard/dbmi/avillach/domain/FederatedQueryRequest.java

```diff
@@ -6,117 +6,112 @@
 import java.util.Map;
 import java.util.UUID;
 
-@Schema(name = "QueryRequest", description = "Object containing credentials map under 'resourceCredentials' and query" +
-		" object under 'query'. The query object expectedResultType can be on of the following " +
-		"\"COUNT\", \"CROSS_COUNT\", \"INFO_COLUMN_LISTING\", \"OBSERVATION_COUNT\", \"OBSERVATION_CROSS_COUNT\", \"DATAFRAME\". ",
-		example = "{\n" +
-				"    \"resourceUUID\": \"<RESOURCE UUID>\",\n" +
-				"    \"query\": {\n" +
-				"        \"categoryFilters\": {\n" +
-				"            \"\\\\demographics\\\\SEX\\\\\": [\n" +
-				"                \"female\",\n" +
-				"                \"male\"\n" +
-				"            ]\n" +
-				"        },\n" +
-				"        \"numericFilters\": {\n" +
-				"            \"\\\\demographics\\\\AGE\\\\\": {\n" +
-				"                \"min\": \"0\",\n" +
-				"                \"max\": \"85\"\n" +
-				"            }\n" +
-				"        },\n" +
-				"        \"requiredFields\": [],\n" +
-				"        \"anyRecordOf\": [\n" +
-				"            \"\\\\demographics\\\\RACE\\\\\"\n" +
-				"        ],\n" +
-				"        \"variantInfoFilters\": [\n" +
-				"            {\n" +
-				"                \"categoryVariantInfoFilters\": {},\n" +
-				"                \"numericVariantInfoFilters\": {}\n" +
-				"            }\n" +
-				"        ],\n" +
-				"        \"expectedResultType\": \"DATAFRAME\",\n" +
-				"        \"fields\": []\n" +
-				"    }\n" +
-				"}")
+@Schema(
+    name = "QueryRequest",
+    description = "Object containing credentials map under 'resourceCredentials' and query"
+        + " object under 'query'. The query object expectedResultType can be on of the following "
+        + "\"COUNT\", \"CROSS_COUNT\", \"INFO_COLUMN_LISTING\", \"OBSERVATION_COUNT\", \"OBSERVATION_CROSS_COUNT\", \"DATAFRAME\". ",
+    example = "{\n" + "    \"resourceUUID\": \"<RESOURCE UUID>\",\n" + "    \"query\": {\n" + "        \"categoryFilters\": {\n"
+        + "            \"\\\\demographics\\\\SEX\\\\\": [\n" + "                \"female\",\n" + "                \"male\"\n"
+        + "            ]\n" + "        },\n" + "        \"numericFilters\": {\n" + "            \"\\\\demographics\\\\AGE\\\\\": {\n"
+        + "                \"min\": \"0\",\n" + "                \"max\": \"85\"\n" + "            }\n" + "        },\n"
+        + "        \"requiredFields\": [],\n" + "        \"anyRecordOf\": [\n" + "            \"\\\\demographics\\\\RACE\\\\\"\n"
+        + "        ],\n" + "        \"variantInfoFilters\": [\n" + "            {\n"
+        + "                \"categoryVariantInfoFilters\": {},\n" + "                \"numericVariantInfoFilters\": {}\n"
+        + "            }\n" + "        ],\n" + "        \"expectedResultType\": \"DATAFRAME\",\n" + "        \"fields\": []\n" + "    }\n"
+        + "}"
+)
```

Comment:
I think the formatter made things worse here. Not sure if we can add some syntax that allows us to skip formatting a section.